lab_4_fill_words_by_ngrams/main.py

```diff
@@ -28,6 +32,19 @@ def _tokenize(self, text: str) -> tuple[str, ...]:  # type: ignore
         Raises:
             ValueError: In case of inappropriate type input argument or if input argument is empty.
         """
+        if not isinstance(text, str) or not text:
+            raise ValueError('WordProcessor._tokenize: Incorrect input')
```

Comment:
remove `WordProcessor._tokenize` from the message

Reply:
done